GiniVision/Classes/OnboardingPage.swift

```diff
@@ -84,9 +84,9 @@ import UIKit
         // Image view
         imageView.translatesAutoresizingMaskIntoConstraints = false
         ConstraintUtils.addActiveConstraint(item: imageView, attribute: .top, relatedBy: .equal, toItem: contentView, attribute: .top, multiplier: 1, constant: 0)
-        ConstraintUtils.addActiveConstraint(item: imageView, attribute: .width, relatedBy: .lessThanOrEqual, toItem: nil, attribute: .width, multiplier: 1, constant: 204)
+        ConstraintUtils.addActiveConstraint(item: imageView, attribute: .width, relatedBy: .lessThanOrEqual, toItem: nil, attribute: .width, multiplier: 1, constant: 612)
```

Comment:
I'm not against the value itself. I'm just wondering why we have to hardcode values like that. My intuition is that a strange value like 204 or 612 is maybe calculated in some way and might be also derived dynamically in code. I'll look into it a little more.